cmd/podman/pod_create.go

```diff
@@ -33,10 +39,27 @@ var podCreateFlags = []cli.Flag{
 		Name:  "name, n",
 		Usage: "Assign a name to the pod",
 	},
+	cli.BoolTFlag{
+		Name:  "pause",
+		Usage: "Create a pause container associated with the pod to share namespaces with",
+	},
+	cli.StringFlag{
+		Name:  "pause-image",
+		Usage: "The image of the pause container to associate with the pod",
```

Comment:
Personally, I am not a big fan of that as those CLI flags have some default value (just set at runtime rather than statically) but the default values aren't displayed in the command's help message which is a bit inconsistent with other flags and it hides the default from the user.

It's not at all a blocker, but maybe we can just reference the exported variables from `libpod` (e.g., `libpod.DefaultInfraImage`) for setting the default?

Reply:
I agree, I think this should be fixed.

Reply:
fixed in both